rad/src/error.rs

```diff
@@ -337,6 +337,19 @@ pub enum RadError {
     /// Invalid reveal serialization (malformed reveals are converted to this value)
     #[fail(display = "The reveal was not serialized correctly")]
     MalformedReveal,
+    // These errors would never appear in the witnet protocol
+    /// Requests that cannot be parsed must always get this error as their result.
+    /// However, this is not a valid result in a Tally transaction, because invalid requests
+    /// are never included into blocks and therefore never get a Tally in response.
+    #[fail(display = "BridgeMalformedRequest")]
+    BridgeMalformedRequest,
```

Comment:
I have my doubts about this being a bridge specific error. There's probably somewhere else where we coult return this and we're returning a more generic error instead.

I'm OK however to leave this as is if we don't see the clear case for using it elsewhere.

Reply:
The problem here is that BridgeMalformedRequest is an error detected by the node that tries to send a request but it obtains an error in validation, not in runtime. So it only happens when it is not the same "person" the requestor and who includes the request

Reply:
any conclusion on this? :)

Reply:
I thought this PR is closed, not sure if we want to implement it because the benefits are small.